doc/contributor-guide/adding-columns.rst

```diff
@@ -10,16 +44,34 @@ Structure
 - Avoid arrays expressed as individual columns (e.g. ``something0``, ``something1``, ``something2``) where possible, as this increases the number of columns drastically (and there is `a limit <https://www.postgresql.org/docs/current/limits.html>`_), makes it hard to query (``SELECT`` clauses need to list all of these individually, and ``WHERE`` clauses may need to include large ``OR`` or ``AND`` conditions), and potentially requires a lot of database storage space.
 - Columns should be named in all lowercase with underscore (``_``) separators, also known as "snake_case".
 
+.. _add-columns-data-sources:
+
 Data sources
 ============
+Possible sources of data include:
+
+- EFD:
+
+  - Determine whether the information is required at the Summit or if the information is required with very low latency (less than a few minutes).
+    In those cases, the Header Service is the recommended source for ConsDB.
+    Arrangements must be made for the data to appear in the headers before ConsDB can store it.
+    Arrangements must also be made with the ConsDB team for the value to be extracted from the headers.
+    This data will go into the ``exposure`` or ``ccdexposure`` table.
+
+  - If the original data source is the EFD but the data is only needed at USDF and can wait for a few minutes before appearing, the Transformed EFD service should be used.
+    This data will go into a ``*_efd`` table.
+
+- A system (e.g. the Camera Control System), service (e.g. Rapid Analysis), or campaign/pipeline (e.g. Nightly Validation) executing at the Summit or USDF.
+  This data will go into a table with a suffix specific to the source.
```

Comment:
Just for my own clarification, RA -> _quicklook, so the suffix is specific, but doesn't have to match the service name. Are there any other suffix specific service/pipeline/system ones yet?